Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -36,29 +44,29 @@ public protocol Router: CLLocationManagerDelegate {
     /**
      Intializes a new `RouteController`.
      
-     - parameter route: The route to follow.
+     - parameter routeResponse: `RouteResponse` object, containing selection of routes to follow.
      - parameter routeIndex: The index of the route within the original `RouteResponse` object.
      - parameter directions: The Directions object that created `route`.
      - parameter source: The data source for the RouteController.
      - parameter tileStoreLocation: Configuration of `TileStore` location, where Navigation tiles are stored.
      */
-    init(along route: Route, routeIndex: Int, options: RouteOptions, directions: Directions, dataSource source: RouterDataSource, tileStoreLocation: TileStoreConfiguration.Location)
+    init(along routeResponse: RouteResponse, routeIndex: Int, options: RouteOptions, directions: Directions, dataSource source: RouterDataSource, tileStoreLocation: TileStoreConfiguration.Location)
```

Comment:
It made sense to say “`along` `route`” in a sentence, but “`along` `routeResponse`” makes less sense. Maybe “`alongRouteAtIndex`, `in` `routeResponse`”, “`routeResponse`, `routeIndex`”, or simply “`indexedRouteResponse`”.